From 794af3f3930602cd33e9517de1fc00c277ddba8f Mon Sep 17 00:00:00 2001
From: Ye Li <ye.li@nxp.com>
Date: Fri, 27 Oct 2017 01:57:44 -0500
Subject: [PATCH 2696/5242] MLK-16709 dts: imx8qm: Disable the UART2 on MEK
 base board

commit  55e167d1392f10558ca18f436368ce066aeaac58 from
https://source.codeaurora.org/external/imx/linux-imx.git

CM4_1 core will use the UART2 on QM MEK base board as its console. Since this port currently
is a backup debug port on A core side, not really used. We disable it in dts to yield the port
for CM4_1.

Signed-off-by: Ye Li <ye.li@nxp.com>
Acked-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
index cabcbd7..30a04009 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
@@ -467,7 +467,7 @@
 &lpuart2 { /* Dbg console */
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_lpuart2>;
-	status = "okay";
+	status = "disabled";
 };
 
 &lpuart3 { /* MKbus */
-- 
1.7.9.5

